### PR TITLE
Figure.velo: Deprecate parameters "color" to "fill" and "uncertaintycolor" to "uncertaintyfill" (remove in v0.12.0)

### PR DESCRIPTION
**Pull request by michaelgrund**
Addresses #1617 

**Description of proposed changes**

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: pygmt/src/velo.py [hunk 1]
```diff
@@ -5,17 +5,24 @@
...
 
 
 @fmt_docstring
+@deprecate_parameter("color", "fill", "v0.8.0", "v0.12.0")
 @use_alias(
     A="vector",
     B="frame",
     C="cmap",
     D="rescale",
     E="uncertaintycolor",
```
Comment (seisman) [inline pygmt/src/velo.py:24]: I'm wondering if we should also deprecated `uncertaintycolor` to `uncertaintyfill`.

Comment (michaelgrund) [reply pygmt/src/velo.py:24]: In my opinion, yes!

Comment (michaelgrund) [reply pygmt/src/velo.py]: Done @seisman.

**Review by seisman — APPROVED** [review]
Looks good to me, but let's see what other @GenericMappingTools/pygmt-maintainers say about the two deprecated parameters.

**Review by weiji14 — APPROVED** [review]
Yep, ok with double deprecation!

**Pull request merged**